jubatus/server/common/signals.cpp

```diff
@@ -77,12 +77,15 @@ jubatus::util::lang::function<void()> action_on_term;
 jubatus::util::lang::function<void()> action_on_hup;
 jubatus::util::concurrent::mutex mutex_on_signal;
 
+void default_action_on_term() {
+  exit(1);
```

Comment:
Invocation of signal handler is one-shot, so we should re-register the signal handler here?